testdata/scripts/config_default.txt

```diff
@@ -0,0 +1,35 @@
+env HOME=$WORK/home
+
+# Gunk generate with out specified in config
+gunk generate ./gunk
+
+# Check that that files were generated in the correct folder
+exists gunk/all.pb.go gunk/all.pb.gw.go
+
+-- go.mod --
+module testdata.tld/util
+
+-- gunk/doc.go --
+package util // make this directory a Go package
+
+-- gunk/util.gunk --
+package util
+
+import (
+    "github.com/gunk/opt/http"
+)
+
+type Message struct {
+	Msg string `pb:"1"`
+}
+
+type Util interface {
+	// Echo echoes a message.
+    //
```

Comment:
mixing tabs and spaces? :) I'd remove the http stuff from these config scripts, it's irrelevant and quite verbose.

Reply:
Arghh, I copied and didn't replace the tabs. The http stuff is required however, as otherwise the `protoc-gen-grpc-gateway` won't produce any output files. And since that is part of the default config, I think we want to test that it works :)